dropwizard-app/src/main/java/org/coner/boundary/CompetitionGroupBoundary.java

```diff
@@ -0,0 +1,34 @@
+package org.coner.boundary;
+
+import org.coner.core.domain.CompetitionGroup;
+
+/**
+ *
+ */
+public class CompetitionGroupBoundary extends AbstractBoundary<
```

Comment:
I don't see a need for it. I can't remember why I introduced it in the first place.